2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,17 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  let averageStock = [];
+  let sum = 0;
+  for (let stockPriceBrand of closingPricesForAllStocks) {
+    for (let unitStockPriceBrand of stockPriceBrand) {
+      sum += unitStockPriceBrand;
+    }
+    let averageAllStock = sum / 5;
```

Comment:
You could use stockPriceBrand.length here instead of 5, to make your code able to cope with new data.

Reply:
I've updated it.